src/develop/Jonny.java

```diff
@@ -0,0 +1,5 @@
+package develop;
+
+public class Jonny {
+    //created jonny
```

Comment:
Hey jonny create a jonny method , don't forget to make it static

Reply:
no thats not right